sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -410,4 +393,85 @@ export class BlobServiceClient extends StorageClient {
       ...options
     });
   }
+
+  /**
+   * Returns an AsyncIterableIterator for ServiceListContainersSegmentOptions
+   *
+   * @private
+   * @param {string} [marker] A string value that identifies the portion of
+   *                          the list of containers to be returned with the next listing operation. The
+   *                          operation returns the NextMarker value within the response body if the
+   *                          listing operation did not return all containers remaining to be listed
+   *                          with the current page. The NextMarker value can be used as the value for
+   *                          the marker parameter in a subsequent call to request the next page of list
+   *                          items. The marker value is opaque to the client.
+   * @param {ServiceListContainersSegmentOptions} [options] Options to list containers operation.
+   * @returns {AsyncIterableIterator<Models.ServiceListContainersSegmentOptions>}
```

Comment:
type is wrong in @returns.
also need to document @returns

Reply:
hmm, no doc is probably fine for private functions.

Reply:
Good catch.